integration/proxy/teleterm_test.go

```diff
@@ -139,17 +139,23 @@ func testGatewayCertRenewal(ctx context.Context, t *testing.T, params gatewayCer
 
 	tshdEventsService := newMockTSHDEventsServiceServer(t, tc, params.inst, params.username)
 
+	webauthLoginCalls := 0
 	webauthnLogin := func(ctx context.Context, origin string, assertion *wantypes.CredentialAssertion, prompt wancli.LoginPrompt, opts *wancli.LoginOpts) (*proto.MFAAuthenticateResponse, string, error) {
 		t.Helper()
+		webauthLoginCalls++
```

Comment:
I should be fine without using `sync/atomic` here, shouldn't I?

If there was a data race, the detector would catch it. So if someone in the future modifies the MFA code to make concurrent calls for some reason, it'll trigger the data race detector.

Reply:
How wrong I was, I ran the tests myself with `-race` and I already stumbled into this. 🤦